### PR TITLE
Ds 3465

### PR DESCRIPTION
**Pull request by TimBock**

### CODE REVIEW

**Comment by TimBock** [comment]
This blocks some work for core: RS-8148

File: R/calibrate.R [hunk 1]
```diff
@@ -124,6 +124,8 @@ convertToDataFrame <- function(x)
 categoricalTargets <- function(adjustment.variables, categorical.targets, subset)
 {
     targets = list()
+    if (missing(subset))
+        subset <- rep(TRUE, nrow(adjustment.variables))
```
Comment (mwmclean) [inline R/calibrate.R:128]: Should this be `NROW` as in numericTargets (Line 179)?

**Review by mwmclean — CHANGES_REQUESTED** [review]

File: R/weightingdialog.R [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+#' @param categorical.targets The target probabilities for each category listed in \code{categorical.variables}
+#' @param numeric.variables An optional list or data frame of categorical adjustment variables.
+#' @param numeric.targets the target mean for each numeric variable in numeric.variables
+#' @param lower A lower bound weight value (not guaranteed to be achieved).
+#' @param upper An upper bound weight value (not guaranteed to be achieved).
+#' @param calfun The calibration function: Raking (Default), Linear, Quadratic.
+#' @param input.weight An optional weight variable; if supplied, the created weight is created to be as close
+#' to this input.weight as possible
+#' @param force.to.n Force the sum of weights to equal the sample size.
+#'
```
Comment (mwmclean) [inline R/weightingdialog.R:13]: Extra whitespace.

File: R/weightingdialog.R [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+#' @return numeric A vector of weights
+#' @export
+
```
Comment (mwmclean) [inline R/weightingdialog.R:16]: Can we add more `@details` to further explain the differences between this function and `Calibrate`?

File: R/weightingdialog.R [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+WeightingDialog <- function(categorical.variables = NULL,
```
Comment (mwmclean) [inline R/weightingdialog.R:18]: Will this code ever be viewed/edited in the R Code editor in Displayr? If so, is there a better name for this function without the word "Dialog"? (Similarly, for `computeWeightsDialog`).

**Review by mwmclean — APPROVED** [review]

**Pull request merged**